### ISSUE TITLE
Does «sai» refer to a type of material or an instance of a material?

### ISSUE THREAD

**Issue by acotis** [toaq/dictionary#100]
For example, is gold a **sai** or is a piece of gold a **sai**?

**Comment by xorxes** [comment]
I would say they're both **sai**, as opposed to time or a chunk of time which are not, for example. I don't see how something could be **sai** and a chunk of it not be, or vice-versa.

**Comment by xorxes** [comment]
I think what you may be wondering though is if you have three pieces of gold and five pieces of silver then how many **sai** do you have. I would say you have two **sai** in that case.

**Comment by acotis** [comment]
Why not ten **sai**? Three pieces of gold (3), five pieces of silver (5), Gold itself (1), and Silver itself (1)?

The difference is also exemplified by the English word "animal", which conflates two similar meanings. If the question is "what's your favorite animal?" then "cats" is a valid answer, but "my cat Simon who is orange and who we've had for six years" probably isn't, even though Simon is undeniably an animal in other contexts. The question was asking about animal _types_.

This also relates to my question about **shijue**, whether it should mean `___ is a January` or `___ is January`. In fact, the underlying word **jue** could mean either `___ is a month [such as July of 1421]` or `___ is a much [such as July]`. Given Hoemai's answer to the **shijue** question, I think it means the former.ȷue

I think in each case, there should be a root word that has only one of the two meanings, and a derived word that has the other meaning. I think I've proposed **zy** for the purpose of converting from individuals to types, so maybe you could have eight **sai** and two **saizy** in that scenario. I also wonder whether **guo** is intended to convert the other way...

**Comment by xorxes** [comment]
I think mixing levels of abstraction will always be wrong, so either two **sai** or eight **sai** depending on the context.

Would mammals qualify as animal-**guo** though, or do you need to go "all the way down"?

I don't think it's possibe to force level of abstraction be part of the predicate, and even if you try, in some cases you can have many more than two levels. Consider for example "word".  How many words are in: "cat, cat, cat"? What if we count the ones you are seeing on your screen and the ones I am seeing on mine?

**Comment by acotis** [comment]
> Would mammals qualify as animal-**guo** though, or do you need to go "all the way down"?

This is a good question. I think I would say **rioq baq [mammal] baq niai**, and I would *also* say **rioq baq kato baq niai**. In other words, I would give both Mammal and Cat the same status of "sub-concept of Animal", but I wouldn't use **guo** for either. I would reserve that for individual animals, actual creatures that walk or swim or fly.

By the way, I do think that "mammals" sounds like a strange answer to the question "what's your favorite animal?". But I think it sounds strange only because my culture comes with some pre-conceived ideas about which types of animals are "basic". Cats yes, mammals no, fish yes, vertebrae no. But then I imagined asking a zoologist what their favorite type of animal is, and I can see anything from "amphibians" to "*Betta splendens*" being the answer. I think the distinction between "basic" and "unbasic" sub-concepts can be tossed out, and then I think there are only two levels: abstract and concrete. **rioq** can be used to compare two abstract things, and **guo** can be used to say a concrete thing is an instance of an abstract one.

I see what you're getting at with the "word" example. Some possible distinctions:

- Each entry in a dictionary is a different word.
- Each numbered definition in the dictionary entry is a different word.
- Each alternative spelling in the dictionary entry is a different word.
- Each written instance is a different word.
- Each spoken instance is a different word.

But I don't view these distinctions as being arranged in a line, with each one more specific than the last. I don't see them as "levels". I see them as being five (or maybe four) different things:

- Words
- Lexemes
- Etymemes? (not sure what the hell to call these)
- Written word instances
- Spoken word instances

So I would agree with you that there are more than two things here, and that the **zy** suffix system is probably unsuited to the task, but I don't see the issue with simply coining five predicates to mean these five things.

**Comment by xorxes** [comment]
> actual creatures that walk or swim or fly

That seems to be saying that mammals are not actual creatures that walk or swim or fly.

I don't see a problem with coining more and more predicates for more and more specific meanings. The problem I see is if you lose the less specific ones in the process. If **shu toa** is a dictionary entry, hopefully you can still say **choa ru kai jí shú toa**, right?

**Comment by acotis** [comment]
> That seems to be saying that mammals are not actual creatures that walk or swim or fly.

It's saying that Mammal, the category, isn't an actual creature (categories are not creatures).

> If **shu toa** is a dictionary entry, hopefully you can still say **choa ru kai jí shú toa**, right?

I think **choa** and **kai** could be defined in such a way that they accept dictionary entries (among other things). But I don't understand the "more and more specific" part. Imagine if in the future, "resume" becomes widely accepted as an alternative spelling of "résumé". The dictionary might have an entry like this:

**resume**: (1) to continue doing something after a pause; (2) a document listing one's qualifications and experience.

But a linguist compiling a list of lexemes and their allowable renditions into letters might include an entry that reads:

DOCUMENT LISTING ONE'S QUALIFICATIONS AND EXPERIENCE: (1) resume, (2) résumé.

Here, "different dictionary entry" and "different lexeme" are not related by the relation "X is a more specific distinction than Y". There are three objects under consideration:

1. resume (to continue)
2. resume (the document)
3. résumé (the document)

And the former classification groups them as (1 2 | 3) while the latter groups them as (1 | 2 3).

The reason I say all this is that I don't really understand: 

> The problem I see is if you lose the less specific ones in the process.

Because to me, these concepts (words, lexemes, etc) aren't related except for the common theme of being about language.

**Comment by xorxes** [comment]
Yeah, I don't think the relationship between written word and spoken word is similar to that between resume and résumé. If you don't have a word in Toaq that covers both then you have lost something in your overspecification. I'm pretty sure **toa** must be true both of written words and of spoken ones though.

**Comment by acotis** [comment]
Do you think there should be a Toaq predicate that applies both to spoken words ("cat cat" is two of these) and dictionary entries ("cat cat" is one of these instantiated twice)?

**Comment by xorxes** [comment]
I do, yes. But this applies to almost any predicate. What counts as something that satisfies it is very often context dependant.

I'm not sure how you can tell whether "cat cat" are two spoken words, one spoken word intantiated twice, two written words, or one written word instantiated twice.

**Comment by acotis** [comment]
Oops, I should have said "two written words". I don't know what "one written word instantiated twice" would mean -- by "written word" I mean "item appearing in a document between two whitespace characters" or something like that.

That, plus a few other things you've said in this discussion, make it sound like there's some general theory of Types & Instances that can be applied to any predicate. But that seems strange to me.

Let's take the two things that people might mean by "animal": one is "an individual living creature" and the other is "a concept X such that every X is an animal (but not necessarily vice versa)".

Would you say that there is an analogous pair of meanings for (almost) any predicate? What would be the two meanings for **reo**, "color"? Or **poq**, "person"?

**Comment by xorxes** [comment]
I disagree that there are two meanings of "animal" involved. Defining "animal" as a concept seems weird.

And you can say "this is my favorite color" (pointing to something concrete) or "a secondary color is made by mixing two primary colors" at a different level of abstraction. Or "this person is friendly" vs "people can be weird sometimes".

**Comment by acotis** [comment]
Aha! Okay, what you just said about the word "animal" is acutely strange to me, which means it might be the source of all other confusion.

What are some things you might call animals? For example, which of the following things are animals?

- My cat Simon
- A bug that just landed on my hand
- Every pig that has ever lived
- _Felis catus_
- _Equus ferus caballus_ (horses)
- _Carnivora_ (carnivors)

And, should these all be **niai**?

**Comment by xorxes** [comment]
I would say the first three, plus horses and carnivors, are all **niai**. I'm not sure about the latin names, it's not something for everyday language anyway.

**Comment by acotis** [comment]
The reason I gave the latin names is that I have the impression that using latin names and italic letters means you're definitely naming a _species_, and not an individual animal. So I actually want to ask for clarification: when you say horses are **niai**, do you mean only that anything that's a horse is also a **niai**, or do you mean that the _species_ "Horse" is a **niNıaı

**Comment by xorxes** [comment]
Would you say that the species is an animal? When you ask what your favorite animal is, you are not asking what your favorite species is. The answer could be "Cats, because they catch mice", or something like that. The reason someone might like cats is probably not because of properties that the species might have, right?

**Comment by acotis** [comment]
Well, it's true you're not asking for a favorite species. After all (despite what I said in my most recent post), Horse is actually *not* a species, it's a *sub*species, one level down in the taxonomy; yet "horses" is still a valid answer to the question "what is your favorite animal?".

But Cat and Horse are both kinds of animals, and I'd say that when you ask "what's your favorite animal?" you're expecting a kind-of-animal as an answer. So I would argue that, in English, yes, Horse is an animal (despite not being an individual animal — types are not individuals).

But my key opinion here is: *it's dumb that English works this way*. It's not as dumb as, say, the word "set" having a whole handful of unrelated meanings, but it's still dumb, IMO. I view it as ambiguity to have the same word mean either individual-animal or type-of-animal.

**Comment by xorxes** [comment]
My point is that there's nothing special about "animal" in this respect. "Chair" can mean individual chair or type-of-chair. "Horse" can mean individual horse or type-of-horse, "Shoe" can mean individual shoe or type-of-shoe, and so on.

**Comment by acotis** [comment]
Hmm, it does seem to be more prolific than I thought. "Which paper do you want?" "construction paper" and so on. But do you think this ambiguity should be imported into Toaq predicates?

**Comment by xorxes** [comment]
I don't really see it as an ambiguity, just different levels of abstraction.
